File: nga/app/src/main/java/org/gosky/nga/view/ForumView.java
package org.gosky.nga.view;

import org.gosky.base.mvp.BaseView;
import org.gosky.nga.data.entity.ThreadBean;

import java.util.List;

/**
 * Created by guozhong on 16/10/24.
 */

public interface ForumView extends BaseView {
    void refreshRcv(List<ThreadBean> str);
    void loadMoreRcv(List<ThreadBean> list);
}

File: nga/app/src/main/java/org/gosky/nga/view/ForumView.kt
package org.gosky.nga.view

import org.gosky.base.mvp.BaseView
import org.gosky.nga.data.entity.ThreadBean

/**
 * Created by guozhong on 16/10/24.
 */

interface ForumView : BaseView {
    fun refreshRcv(str: List<ThreadBean>)
    fun loadMoreRcv(list: List<ThreadBean>)
}
